### ISSUE TITLE
Using `<Tab>` key in `git_status` picker moves the selector to the bottom

### ISSUE THREAD

**Issue by ZenithFlux** [nvim-telescope/telescope.nvim#3002]
### Description

Whenever `<Tab>` key is pressed to stage a file in `git_status` picker, the selector moves to the bottom of the window. This makes staging multiple files cumbersome as you have to move up each time you stage.

### Neovim version

```markdown
NVIM v0.9.5
Build type: RelWithDebInfo
LuaJIT 2.1.1692716794
```

### Operating system and version

Linux Mint 21.3

### Telescope version / branch / rev

telescope 1.6

### checkhealth telescope

```markdown
telescope: require("telescope.health").check()

Checking for required plugins ~
- OK plenary installed.
- OK nvim-treesitter installed.

Checking external dependencies ~
- OK rg: found ripgrep 14.1.0 (rev e50df40a19)
- WARNING fd: not found. Install [sharkdp/fd](https://github.com/sharkdp/fd) for extended capabilities

===== Installed extensions ===== ~

Telescope Extension: `fzf` ~
- OK lib working as expected
- OK file_sorter correctly configured
- OK generic_sorter correctly configured
```

### Steps to reproduce

1. Open `telescope.builtin.git_status()` picker.
2. Move up a few files in the picker.
3. Press `<Tab>`.

### Expected behavior

The selector should not move when staging or unstaging files in the `git_status` picker.

### Actual behavior

Selector jumps to the bottom of the picker when `<Tab>` is pressed to stage the file.

### Minimal config

```Lua
local lazypath = vim.fn.stdpath 'data' .. '/lazy/lazy.nvim'
if not vim.loop.fs_stat(lazypath) then
  vim.fn.system {
    'git',
    'clone',
    '--filter=blob:none',
    'https://github.com/folke/lazy.nvim.git',
    '--branch=stable',
    lazypath,
  }
end
vim.opt.rtp:prepend(lazypath)

require('lazy').setup({{
    'nvim-telescope/telescope.nvim',
    branch = '0.1.x',
    }}, {})
```

**Comment by andtankian** [comment]
Same behavior here, the input disappears as well (everytime the <Tab> - add to stage) is pressed.

**Comment by andtankian** [comment]
It was fixed probably in c0ee29e, however it seems using tag v0.1.6 is not syncing those fixes.

To get it working here I just removed the `tag` property from lazy.

```lua
{
    'nvim-telescope/telescope.nvim',
    -- tag = "0.1.6",
    dependencies = { 'nvim-lua/plenary.nvim' }
 }
 
 It's important to notice that by doing this may occurs breaking changes updates.

**Comment by ZenithFlux** [comment]
@andtankian i checked out the latest commit and looks like they have fixed it. I think the fix will be applied when v1.7 will release. You can checkout the latest commit at your own risk since it is not officially released yet.

I have kept this issue open so that anyone who is having this problem have a common place to talk about it.

**Comment by ZenithFlux** [comment]
> It was fixed probably in c0ee29e, however it seems using tag v0.1.6 is not syncing those fixes.

Yeah same, 😶

**Comment by jamestrew** [comment]
Thanks guys, I've flagged the related PR(s) to be backported to the 0.1.x branch for the next release (0.1.7).
No ETA but we try to do a release every few months so maybe in roughly another month.

I'll close this issue since this is already fix on master.
